geometry_msgs/Point[] point_cloud
---
geometry_msgs/Point[] path